### DISCUSSION TITLE
post image flickering

### DISCUSSION THREAD

**Discussion by safinakib** [cotes2020/jekyll-theme-chirpy#1178] [Q&A]
i don't seem to understand why my post image is flickering
down below im providing site link and markdown link
https://safinakib.github.io/jekyll-theme-chirpy/posts/Aamrupali/
https://github.com/safinakib/jekyll-theme-chirpy/blob/master/_posts/2023-07-22-Aamrupali.md

**Comment by sukhjinder-kumar** [comment]
You forked this repo, try cloning the starter template. This was an issue in previous versions. Got patched up in 6.1.0, haven't noticed since.
